### PR TITLE
Add functionalities in v.0.7.0 specifications

### PR DESCRIPTION
**Pull request by azzamsa**

### CODE REVIEW

**Comment by anak10thn** [comment]
@azzamsa wrong version, 0.0.7->0.7.0

**Comment by azzamsa** [comment]
oh my bad, sorry.

File: api/app/models/model.py [hunk 1]
```diff
@@ -1,197 +1,156 @@
...
     try:
-        a = client.read("/"+stored)
-    except EtcdKeyNotFound as e:
-        raise e
-    else:
-        for child in a.children:
-            data = ast.literal_eval(child.value)
-            result.append(data)
-    return result
+        query = f"SELECT column_name FROM information_schema.columns WHERE table_schema = 'public' AND table_name='{table}'"
```
Comment (anak10thn) [inline api/app/models/model.py]: Please use prepared statement http://initd.org/psycopg/articles/2012/10/01/prepared-statements-psycopg/

File: api/app/models/model.py [hunk 2]
```diff
@@ -1,197 +1,156 @@
...
-    try:
-        record_data = read_by_id("record", key)
-    except Exception as e:
-        raise e
+        connection.commit()
+        rows_deleted = cursor.rowcount
+        return str(rows_deleted)
+
+
+def retry_execute(query, column, retry_counter, error):
```
Comment (anak10thn) [inline api/app/models/model.py]: what this's?

File: api/app/models/model.py [hunk 3]
```diff
@@ -1,197 +1,156 @@
...
+    limit_retries = 5
+    results = []
+    cursor, connection = get_db()
+    if retry_counter >= limit_retries:
+        raise error
     else:
+        retry_counter += 1
+        print("got error {}. retrying {}".format(str(error).strip(), retry_counter))
```
Comment (anak10thn) [inline api/app/models/model.py]: can you use logger exception for std error msg? https://docs.python.org/3/library/logging.html#logging.Logger.exception

**Pull request merged**